### DISCUSSION TITLE
Example

### DISCUSSION THREAD

**Discussion by mustafa-archived** [catppuccin/catppuccin#2033] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

Stylus extension

### Any additional comments?

Work can be found [here](https://github.com/mustafakhalaf-git/example/)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by kkrypt0nn** [comment]
Taking into consideration the main goal of ports creation being

> A port is an adaptation of Catppuccin's palette **for an app to use.**

The https://example.com website has no use for an average user and no sane person will go on a recurring basis on said website. For that reason I believe it should not be added into the organization, though if you want to keep it on your private user why not.

**Reply by mustafa-archived** [reply]
closed it

**Reply by vanillyn** [reply]
though to be fair, it could be used as a literal example for a userstyle theme
it does check everything

**Reply by mustafa-archived** [reply]
I made this port as a joke, but maybe

I'll mention @sgoudham and see what he thinks

**Reply by kkrypt0nn** [reply]
That could be an idea, though I'd rather spend some more time and use a website where you can style a bit more than just a `body`, `div` and `a` tag selector and 90% of the colors are not used. A basic example website made by us with the most used  HTML elements, e.g. drop-downs, buttons, hyperlinks, etc. could definitely be well enough.

That way it may actually serve as an example on how websites could be themed.
